Library/Ultra Tech/Ultra Tech Equipment.eqp

```diff
@@ -18037,28 +18025,233 @@
 			}
 		},
 		{
-			"id": "elI6n8v2PinZcDSub",
+			"id": "eLSIMa6gfZB_ag_HF",
```

Comment:
The ID of Gripboots (TL9) has been changed, which may break existing synchronization. Is this intentional? I can see how this change could be warranted: It forces those seeking re-synchronisation to re-examine whether they have the right TL variant.